File: 24HReportSystemData/Models/AccountInfo.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;

#nullable disable

namespace _24HReportSystemData.Models
{
    public partial class AccountInfo
    {
        [JsonIgnore]
        public string AccountId { get; set; }
        public string Fullname { get; set; }
        public string Address { get; set; }
        public string IdentityCard { get; set; }

        [JsonIgnore]
        public virtual Account Account { get; set; }
    }
}

File: lib/entities/account_info.dart
class AccountInfo {
  AccountInfo({
    required this.fullname,
    required this.address,
    required this.identityCard,
    // required this.isAuthen,
  });
  String fullname;
  String address;
  String identityCard;
  // bool isAuthen;

  factory AccountInfo.fromJson(Map<String, dynamic> json) => AccountInfo(
        fullname: json["fullname"] ?? '',
        address: json["address"] ?? '',
        identityCard: json["identityCard"] ?? '',
        // isAuthen: json["isAuthen"] ?? false,
      );

  Map<String, dynamic> toJson() => {
        "fullname": fullname,
        "address": address,
        "identityCard": identityCard,
        // "isAuthen": isAuthen,
      };
}
